rentals_book.rb

```diff
@@ -0,0 +1,42 @@
+class Rents
+  def initialize
+    @person_array = []
+    @books = []
+    @rental_array = []
+  end
+
+  def create_rental
+    puts 'Select a book from the following list by number: '
+    @books.each_with_index do |book, index|
+      puts "#{index}) #{book[:output]}"
+    end
+    book_selected = Integer(gets.chomp)
+    book_chosen = @books[book_selected][:object]
+
+    puts 'Select a person from the following list by number (not id): '
+    @person_array.each_with_index do |person, index|
+      puts "#{index}) #{person[:output]}"
+    end
```

Comment:
- [ ] There seems to be something wrong with your app. When I try to create a rental, I do not get the list of books I created initially. And from what I can see (in lines 3 and 17) I also would not be able to get the list of people. This is because the books and people you created with the BooksList and Methods classes are not accessible here. Hence the arrays you are trying to iterate will always be empty. Kindly fix this.

  **SUGGESTION:** You could probably resolve this by passing in instances of BookList and Methods as arguments in the rent instance in main.rb and then accessing their respective arrays.

  Also, note that your app is expected to work as in the **Add Basic UI** project

  ![ngu-create](https://user-images.githubusercontent.com/36542986/149897114-82f9c075-8078-48e2-81da-5b6adef2dd25.jpg)